src/lib/connectors/jsdom/jsdom.ts

```diff
@@ -77,6 +77,7 @@ class JSDOMConnector implements IConnector {
     private _manifestIsSpecified: boolean = false;
     private _window: Window;
     private _document: JSDOMAsyncHTMLDocument;
+    private _fetchedHrefs: Array<string>;
```

Comment:
I think this is perfect for a `Set` instead of an array.

Reply:
@sarvaje Fixed.